spyvm/model.py

```diff
@@ -1035,6 +1035,13 @@ def short_atput0(self, space, index0, w_value):
     def size(self):
         return self._size
 
+    def as_string(self):
```

Comment:
So just say jit.look_inside_if(lambda self: jit.isconstant(self._size)) or sth like that